test/prism/encoding_test.rb

```diff
@@ -54,7 +55,8 @@ class EncodingTest < TestCase
       Encoding::CP51932 =>      0x00...0x10000,
       Encoding::GBK =>          0x00...0x10000,
       Encoding::Shift_JIS =>    0x00...0x10000,
-      Encoding::Windows_31J =>  0x00...0x10000
+      Encoding::Windows_31J =>  0x00...0x10000,
+      Encoding::Windows_874 =>  0x00...0x10000
```

Comment:
The maximum codepoint for `Windows_874` is `0xFF`, so we should put this up in the hash with the other 1-byte encodings.